src/components/Overlay/Overlay.tsx

```diff
@@ -29,14 +35,20 @@ const Overlay: FC<OverlayProps> = ({
   onClose,
   title = "",
   isHidden = true,
+  subTitle = "",
   children,
 }) => {
   const shouldReduceMotion = useReducedMotion();
 
   return (
     <Container hasTitle={!!title} isHidden={isHidden}>
       <TitleContainer>
-        <StyledTitle type="h2">{title}</StyledTitle>
+        <TitleSubContainer>
+          <StyledTitle type="h2">{title}</StyledTitle>
+          {!!subTitle && !isHidden && (
```

Comment:
Removing it when `isHidden` will warp the close animation. This should be animated with css, just like the title.